pmoves/services/replay-compressor/app.py

```
@@ -0,0 +1,282 @@
+    endpoint_url=S3_ENDPOINT_URL,
+    aws_access_key_id=S3_ACCESS_KEY,
+    aws_secret_access_key=S3_SECRET_KEY,
+    region_name=S3_REGION,
+    config=Config(signature_version="s3v4"),
+)
+
+
+def ensure_bucket() -> None:
+    try:
+        s3.head_bucket(Bucket=S3_BUCKET)
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)